packages/web-app-files/src/components/SideBar/Shares/Links/ListItem.vue

```diff
@@ -33,7 +33,7 @@
           name="lock-password"
           class="oc-files-file-link-has-password oc-mr-xs"
           fill-type="line"
-          :aria-label="$gettext('This link is password-protected')"
```

Comment:
This is causing the failure in the tests. The icon is no longer labelled even though it has role `img`.